api/db-cdr/generate-cdr/run-achilles-queries.sh

```diff
@@ -382,6 +411,30 @@ COUNT(distinct p1.PERSON_ID) as source_count_value from \`${BQ_PROJECT}.${BQ_DAT
 where co1.procedure_concept_id != co1.procedure_source_concept_id
 group by co1.procedure_source_concept_id,p1.gender_concept_id"
 
+#  3107 Gender identity
```

Comment:
How does this differ from the above query? COuld you add a comment to this one as well?

Reply:
this calculates the gender identity counts for procedure concepts.